File: satoriviacafe-common/src/main/java/com/satoriviacafe/common/utils/valki/I18nUtil.java
package com.satoriviacafe.common.utils.satoriviacafe;

import org.springframework.context.i18n.LocaleContextHolder;

/**
 * @author shy
 * @since 2025年05月17日
 */
public class I18nUtil {
    /**
     * 获取当前语言代码
     * @return 语言代码
     */
    static public String getLangCode() {
        return LocaleContextHolder.getLocale().toLanguageTag();
    }

}

File: satoriviacafe-common/src/main/kotlin/com/satoriviacafe/common/utils/I18nUtil.kt
package com.satoriviacafe.common.utils

infix fun String.i18n(arg: Any?): String = MessageUtils.message(this, arg)
infix fun String.i18n(args: Array<out Any?>): String = MessageUtils.message(this, *args)
inline val String.i18n: String get() = MessageUtils.message(this)

